json/ubuntu-24.04.json

```diff
@@ -0,0 +1,8 @@
+{
+    "uuid": "ff1f3e17-84d9-4d72-962d-571c4ceb527b",
+    "reference_label": "ubuntu-24.04",
+    "name_label": "Ubuntu Noble Numbat 24.04",
+    "derived_from": "base-hvmlinux.json",
+    "min_memory": "1G",
```

Comment:
Did you have a test with the default value like memory with 1G and disk with 10G and ubuntu 24.04 is installed successfully?

Reply:
Sure, I verified on the XS 8, there is no significant gap between X8 and Yangtze. if this kind configuration worked on XS8, it should work on Yangtze as well